pyiron_contrib/atomistic/atomicrex/potential_factory.py

```diff
@@ -8,50 +8,56 @@
 
 class ARPotFactory(PyironFactory):
     @staticmethod
-    def eam_potential(identifier="EAM", export_file="output.eam.fs", rho_range_factor=2.0, resolution=10000, species=["*", "*"]):
+    def eam_potential(identifier="EAM", export_file="output.eam.fs", rho_range_factor=2.0, resolution=10000, species=["*","*"]):
         return EAMPotential(
             identifier=identifier,
             export_file=export_file,
             rho_range_factor=rho_range_factor,
             resolution=resolution,
             species=species,
-            )
-    
+        )
+
     @staticmethod
     def lennard_jones_potential(sigma, epsilon, cutoff, species={"a": "*", "b": "*"}, identifier="LJ"):
         return LJPotential(sigma, epsilon, cutoff, species=species, identifier=identifier)
 
-class LJPotential():
-    def __init__(self, sigma, epsilon, cutoff, species, identifier):
+
+class AbstractPotential(InputList):
+    def __init__(table_name="potential"):
```

Comment:
Missing `self` argument.

Reply:
Yes, I fixed this in https://github.com/pyiron/pyiron_contrib/pull/72/commits/c196a7f73be9b13d8c5d1b08313f66aff74be0f5